etcdctl/ctlv2/command/util.go

```diff
@@ -78,7 +79,7 @@ func getPeersFlagValue(c *cli.Context) []string {
 
 	// If we still don't have peers, use a default
 	if peerstr == "" {
-		peerstr = "http://127.0.0.1:2379,http://127.0.0.1:4001"
```

Comment:
Just need a final confirmation from you on whether to keep os specific seperation or keep it common?